### ISSUE TITLE
Metadata block interference patterns

### ISSUE THREAD

**Issue by simonrharris** [jameshadfield/phandango#122]
When you have big trees with large blocks of the same colour metadata, interference patterns emerge. This seems to be caused by tiny white lines between the blocks. I wonder if this would be solved by rounding all coordinates before drawing the blocks so that they are full integers?

**Comment by jameshadfield** [comment]
yeah i think that would be a good solution. I'm not sure if the boxes are trimmed or there are white lines drawn
